MolecularNodes/md.py

```diff
@@ -107,8 +107,34 @@ def load_trajectory(file_top,
     
     
     # determin the bonds for the structure
+    # if hasattr(univ, 'bonds') and include_bonds:
```

Comment:
You can delete these lines which are commented out. If the bug fix is adopted, they shouldn't be kept in the file